www/include/configuration/configObject/service_dependency/getServiceXml.php

```diff
@@ -117,17 +117,19 @@
 					WHERE hsr.host_host_id IS NULL 
                                         AND hsr.hostgroup_hg_id IN (SELECT hostgroup_hg_id 
                                                                     FROM hostgroup_relation 
-                                                                    WHERE host_host_id = '" . $db->escape($hostId). "') 
-                                        AND h.host_id = '" . $db->escape($hostId). "' 
+                                                                    WHERE host_host_id = :hostId ) 
+                                        AND h.host_id = :hostId 
                                         AND s.service_id = hsr.service_service_id 
                                         AND s.service_register = '1' 
                                 ) AS res $aclFrom $aclCond
 				ORDER BY res.host_name, res.service_description";
     }
-    $res = $db->query($query);
-    while ($row = $res->fetchRow()) {
+    $stmt = $db->query($query);
+    $stmt->bindValue(':hostId', $hostId, PDO::PARAM_INT);
+    $stmt->execute();
+    while ($row = $stmt->fetchRow()) {
```

Comment:
```suggestion
    while ($row = $stmt->fetch(\PDO::FETCH_ASSOC)) {
```

Reply:
@callapa -> as discussed, the fetchRow() method is called in a while loop, it won't work with a FETCH_ASSOC